packages/renderer/src/lib/pvc/pvc-utils.ts

```diff
@@ -29,13 +29,14 @@ export class PVCUtils {
     // PVC's only have "Pending", "Bound" and "Lost" as phases.
     // However, we can also determine if it is in the middle of deleting by
     // checking if the phase is "Terminating" through checking if the object has deletionTimestamp.
+    //
+    // Note: If it is "Pending" it will be considered as "STOPPED" as it is not yet bound, this
+    // is similar to how we handle Deployment statuses.
```

Comment:
I would prefer I think to have the else if entry, just assigning to status = 'STOPPED' in the else condition.

Reply:
it is by default set to `let status = 'STOPPED';` (see line 35), so didn't need to add an if statement.

We could add Pending, but  need to make changes to UI component as there is no "PENDING" only starting.